example/src/main/tut/typeclass/Monoid.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: docs
+title:  "Monoid"
+---
+
+# Monoid
+
+**Category Theory**
+
+*A monoid is a semigroup with a unique identity element.*
+
+A monoid instance must satisfy the following laws in addition to those defined by [Semigroup](Semigroup.html):
+
+- `append(empty, x) === x`
+- `append(x, empty) === x`
+
+**Typical imports**
+```tut:silent
+import scalaz.typeclass.Monoid
+```
+
+## Instance declaration
+
+```tut
+import scalaz.typeclass.Semigroup
+import scalaz.typeclass.Semigroup._
+
+def ListSemigroup[A]: Semigroup[List[A]] = new Semigroup[List[A]] {
+  def append(a1: List[A], a2: => List[A]) = a1 ++ a2
+}
+
+implicit def ListMonoid[A]: Monoid[List[A]] = new Monoid[List[A]] {
+  val semigroup = ListSemigroup[A]
+  val empty = List.empty[A]
+}
```

Comment:
Mentioning `MonoidClass` (which lets you define `append` and `empty` instead) might be useful as well.

Reply:
I added an additional example